src/modules/dashboard/model/dashboardClasses.ts

```diff
@@ -64,4 +66,11 @@ export enum TipType{
 export enum AmountType{
   RAW = "raw",
   EXACT = "exact"
+}
+
+export class OtherAsset{
+  asset: string = '';
+  amount: number = 0;
+  assetid: MosaicId;
+  isLinked: boolean = false;
```

Comment:
as I said before, you does not need to have asset property for hex ID, you can get it from the assetId. 

You do not have constructor for your class, please use interface if you do not want to create instance of object.

Reply:
Maybe I should rename the asset name, to better reflect the purpose of the property. The asset in string store either the namespace name, or it is not linked to any namespace, will store asset id instead. Is it okay like this?

Reply:
ya, rename it as fullName. please change assetid to assetId, to be consistent. 
What about the constructor ? there is a required property assetid with no default value